Themes/default/Profile.template.php

```diff
@@ -1681,13 +1681,19 @@ function template_profile_theme_settings()
 			continue;
 		elseif ($setting['id'] == 'wysiwyg_default' && !empty($modSettings['disable_wysiwyg']))
 			continue;
-		elseif ($setting['id'] == 'topics_per_page' && !empty($modSettings['disableCustomPerPage']))
-			continue;
 		elseif ($setting['id'] == 'drafts_autosave_enabled' && (empty($modSettings['drafts_autosave_enabled']) || (empty($modSettings['drafts_post_enabled']) && empty($modSettings['drafts_pm_enabled']))))
 			continue;
 		elseif ($setting['id'] == 'drafts_show_saved_enabled' && (empty($modSettings['drafts_show_saved_enabled']) || (empty($modSettings['drafts_post_enabled']) && empty($modSettings['drafts_pm_enabled']))))
 			continue;
 
+		// Some of these may not be set...  Set to defaults here
+		if ($setting['id'] == 'topics_per_page' && !isset($context['member']['options'][$setting['id']]))
+			$context['member']['options'][$setting['id']] = 0;
+		if ($setting['id'] == 'messages_per_page' && !isset($context['member']['options'][$setting['id']]))
+			$context['member']['options'][$setting['id']] = 0;
+		if ($setting['id'] == 'display_quick_mod' && !isset($context['member']['options'][$setting['id']]))
+			$context['member']['options'][$setting['id']] = 0;
+
```

Comment:
Why not simplify it a bit?
```php
$opts = array('topics_per_page', 'messages_per_page', 'display_quick_mod');
if (in_array($setting['id'], $opts) && !isset($context['member']['options'][$setting['id']]))
    $context['member']['options'][$setting['id']] = 0;
```

Reply:
Yes, good idea.  Will get to it if not tonight then tomorrow.

Reply:
Done